src/modules/events-app.module/module.html

```diff
@@ -8,7 +8,30 @@
   }
 %}
 
-<div class="event-registration">
+{%- macro outputColorIfSet(color) -%}
+    {%- if color.color -%}
+      {%- if opacity < 100 -%}
+        rgba({{color.color|convert_rgb}}, {{color.opacity/100}})
+      {%- else -%}
+        {{color.color}}
+      {%- endif -%}
+    {%- else -%}
+      inherit
+    {%- endif -%}
+{%- endmacro -%}
+
+<div class="event-registration"
+     style="background-color:{{outputColorIfSet(module.background_color)}};
+            color:{{outputColorIfSet(module.font_color)}};">
+  <style>
```

Comment:
I think we should add `{% require_css %}` around this

Reply:
Ooh, this sounds interesting. Could I move all these changes into a single CSS file? I know we're moving away from `module.css` files; what might you suggest here?

Reply:
Oh sorry, I meant adding the wrapped around this `style` block. Embedding HubL into scss isn't very straightforward. https://designers.hubspot.com/docs/hubl/tags#require-css

Reply:
I just realized that you meant me to do this, apologies, @anthmatic! I'll make the change now.